Updating hypernuclei implementation in O2

jditzelnew opened the pull request:
The O2 databases for MC were taken from an older AliRoot version and are not up-to-date. This commit updates the masses, lifetimes and decay channels for the hypernuclei.

jditzelnew committed:
Update O2DatabasePDG.h

Adding correct implementations of hypernuclei

jditzelnew committed:
Update O2MCApplication.cxx

Inserting the correct implementation for hypernuclei, their masses, lifetimes and decay channels

jditzelnew committed:
Update O2MCApplication.cxx

Update Intendations

jditzelnew committed:
Delete DataFormats/simulation/include/SimulationDataFormat/O2DatabasePDG.h

jditzelnew committed:
Adding O2DatabasePDG.h after formatting changes

jditzelnew committed:
Delete Steer/src/O2MCApplication.cxx

jditzelnew committed:
Uploading O2MCApplication.cxx after formatting changes

alibuild committed:
Please consider the following formatting changes

jditzelnew committed:
Merge pull request #1 from alibuild/alibot-cleanup-13746

Please consider the following formatting changes to #13746

jditzelnew committed:
Updating Copyrights

jditzelnew committed:
Updating Copyrights

sawenzel reviewed (changes requested):
this PR seems to contains significant formatting changes or removal of comments. I have difficulties seeing the actual changes in physics parameters that this PR provides, which makes the review process harder. Could you please provide a single commit with the least amount of changes possible?

jditzelnew committed:
Removind hypernuclei named twice

jditzelnew commented:
Dear Sascha, Thanks for your comment. Unfortunately, I cannot do a commit containing only the changes due to format issues. The lines that were changed are 285-346 (fixing masses and lifetimes for hypernuclei and inserting Xi-bound states) and 724-1035 (adding the decay channels and Xi-bound states decay channels) in O2MCApplication.cxx, as well as 237-289 (fixing hypernuclei masses) in O2DatabasePDG.h. We also removed the excited states of the A=4 hypernuclei, since these excited states decay electromagnetically to the ground state which will be reconstructed. So only the ground state is needed (see also latest AliRoot implementation in AliMC.cxx). We updated the O2DatabasePDG.h again, since there were two hypernuclei named twice.

jditzelnew committed:
Delete file to upload new one with correct formatting

jditzelnew committed:
Upload new O2DatabasePDG.h with correct formatting using clang-format

alibuild committed:
Please consider the following formatting changes

jditzelnew committed:
Merge pull request #3 from alibuild/alibot-cleanup-13746

Please consider the following formatting changes to #13746

jditzelnew committed:
Update Copyright

sawenzel commented:
> Dear Sascha, Thanks for your comment. Unfortunately, I cannot do a commit containing only the changes due to format issues. The lines that were changed are 285-346 (fixing masses and lifetimes for hypernuclei and inserting Xi-bound states) and 724-1035 (adding the decay channels and Xi-bound states decay channels) in O2MCApplication.cxx, as well as 237-289 (fixing hypernuclei masses) in O2DatabasePDG.h. We also removed the excited states of the A=4 hypernuclei, since these excited states decay electromagnetically to the ground state which will be reconstructed. So only the ground state is needed (see also latest AliRoot implementation in AliMC.cxx). We updated the O2DatabasePDG.h again, since there were two hypernuclei named twice.

I am afraid this PR can not be merged in this form. The PR removes or restructures comments, changes include files or their order and applies other change that have nothing to do with your intented changes. Please open a fresh PR with just the relevant changes of updating physics constants. If there are formatting-issues I can try to fix them separately. A single well-contained commit would be appreciated.
